### DISCUSSION TITLE
Allow specific Progress sections to be animated

### DISCUSSION THREAD

**Discussion by sartak** [mantinedev/mantine#4561] [Feature requests]
Hi, thanks for Mantine! Loving it.

For the [Progress](https://mantine.dev/core/progress/) component, `animated` applies only at the top level. It'd be sweet if I could toggle `animated` at the **section** level, as an affordance to hint which step of some complex process is active, like so:

```typescript
<Progress
  sections={[
    { value: 40, color: 'cyan', label: 'Download' },
    { value: 20, color: 'blue', label: 'Process' },
    { value: 15, color: 'indigo', label: 'Upload', animated: true },
  ]}
/>
```

**Comment by rtivital** [comment]
It will be added in 7.0
